packages/cron/src/jobs/names.js

```diff
@@ -0,0 +1,73 @@
+import debug from 'debug'
+import retry from 'p-retry'
+
+const MAX_REQUEST_PAGE = 10
+const W3NAME_API_URL_STAGING = 'https://w3name-staging.protocol-labs.workers.dev/'
+const W3NAME_API_URL_PRODUCTION = 'https://w3name-production.protocol-labs.workers.dev/'
```

Comment:
```suggestion
const W3NAME_API_URL_STAGING = 'https://name.web3.storage/'
const W3NAME_API_URL_PRODUCTION = 'https://name-staging.web3.storage/'
```

This might be slightly preemptive, as [w3name/#23](https://github.com/web3-storage/w3name/pull/23) isn't merged yet, even if it's not set up this way yet, it will be. So I think it's worth changing our URLs here.